recipes/gmsh/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "gmsh" %}
+{% set version = "3.0.6" %}
+{% set sha256 = "9700bcc440d7a6b16a49cbfcdcdc31db33efe60e1f5113774316b6fa4186987b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}-source.tar.gz
+  url: http://gmsh.info/src/{{ name }}-{{ version }}-source.tgz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  # remove the tests of OpenCascade and fltk until we implement them in conda-forge
+  script: sed -i '1919i \ \ list(REMOVE_ITEM TESTFILES "${CMAKE_SOURCE_DIR}/tutorial/t16.geo")' CMakeLists.txt && sed -i '1919i \ \ list(REMOVE_ITEM TESTFILES "${CMAKE_SOURCE_DIR}/demos/post_processing/lowmem-anim.geo")' CMakeLists.txt && mkdir build && cd build && cmake .. -DCMAKE_INSTALL_PREFIX=$PREFIX && make && make test ARGS="-V" && make install
```

Comment:
You might consider formalizing these as a patch https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#patches

Reply:
I must not have understood...! Is for conda a patch file like a bash file?